mono/mini/driver.c

```diff
@@ -2186,10 +2202,12 @@ mono_main (int argc, char* argv[])
 			mono_jit_set_aot_mode (MONO_AOT_MODE_LLVMONLY_INTERP);
 		} else if (strcmp (argv [i], "--print-vtable") == 0) {
 			mono_print_vtable = TRUE;
-		} else if (strcmp (argv [i], "--stats") == 0) {
+		} else if (strncmp (argv [i], "--stats", 7) == 0) {
```

Comment:
I would actually prefer you keep this as a separate condition. The problem with this code is that I could pass the argument `--statsASDF` and it would still trigger this block because you're using `strncmp`. The actual case where this matters is if we later were to add some other argument that begins with `--stats`: for example, `--stats-extra`.